packages/frontend-v2/components/ScheduleCourse/ScheduleCourse.tsx

```diff
@@ -107,12 +106,32 @@ export const ScheduleCourse = forwardRef<
             alignItems: "center",
             flexGrow: 1,
             background: "",
-            padding: "8px 8px",
+            padding:
+              scheduleCourse.classId === "CO-OP" ? "48px 8px" : "8px 8px",
```

Comment:
Add `flex: 1` to the styling so that it just fills out all the space when it gets put into a term, and keep the same padding when the course is in the sidebar

Reply:
ur mom